docs/examples/install_tendermint.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# get and unpack golang
+curl -O https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz
```

Comment:
this will not work for x86 and macOS... maybe we need an `if OS == darwin`

Reply:
i know, that's explicit in this line https://github.com/tendermint/tendermint/pull/1069/files#diff-efa5271fdabd2250b8d2dd60cfc5ff1dR9

This script is meant explicitly for a Ubuntu on a DO box. I'll make it clearer in the script itself